src/tripCard/TripCard.example.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { WarningIcon } from '../icon/warningIcon'
+import { Address, Itinerary } from '../newItinerary'
+import { TripCard } from '.'
+
+// Needs to be outside of the .mdx file, cause the parser changes object types
```

Comment:
So there are no possibility to play with knobs?

Reply:
Not with SmartKnobs. But we can add manual knobs. It was broken on local storybook